Source/NSLayoutConstraint.m

```diff
@@ -652,16 +653,92 @@ - (void) updateConstraints
 
 @implementation NSView (NSConstraintBasedLayoutInstallingConstraints)
 
+- (void) _bootstrapAutoLayout
+{
+  if ([self window])
+    {
+      [self.window _bootstrapAutoLayout];
```

Comment:
Could you please use `[self window]`here. All compilers support the dot notation, but some programmers don't like  it and mixing the two notations in one line just seems wrong.

Reply:
Fixed